test/integration.tags

```diff
@@ -1,5 +1,7 @@
 @abi
 @algod
+@applications
```

Comment:
Is the `@applications` test necessary? There isn't presently an unqualified `@applications` tag in the SDK testing branch `feature/box-storage`

Reply:
I am adding this tag to make the `integration.tags` uniform with others in other sdks, but if `@applications` is unnecessary, then we should just take them all out cross SDKs.

Reply:
Ahh right. Yeah, I probly should have removed `@applications` when I refactored the repos to use sandbox. We should just make sure the same tests run before/after the tag removal.

Reply:
the tag should gone in latest commit, and the commit should pass

Reply:
test passing, thanks for point out unnecessary tags, I am resolving this thread here